### PR TITLE
Add Spanish dimex100 example

### PR DESCRIPTION
**Pull request by alx741**
Example for Latin American Spanish, based on the "*dimex100*" corpus

### CODE REVIEW

File: egs/dimex100/s5/run.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+########
+# Config
+########
+
+train_cmd="utils/run.pl"
```
Comment (danpovey) [inline egs/dimex100/s5/run.sh]: Can you please use the standard approach of having a cmd.sh which is sourced by this file, e.g.
`. ./cmd.sh` ?

File: egs/dimex100/s5/local/score.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Note: the double level of quoting for the sed command
+$cmd LMWT=$min_lmwt:$max_lmwt $dir/scoring/log/score.LMWT.log \
+   cat $dir/scoring/LMWT.tra \| \
+    utils/int2sym.pl -f 2- $symtab \| sed 's:\<UNK\>::g' \| \
+    compute-wer --text --mode=present \
+     ark:$dir/scoring/test_filt.txt  ark,p:- ">&" $dir/wer_LMWT || exit 1;
+
+# Show results
+for f in $dir/wer_*; do echo $f; egrep  '(WER)|(SER)' < $f; done
```
Comment (danpovey) [inline egs/dimex100/s5/local/score.sh]: Please add a RESULTS file to show what the expected results are.

File: egs/dimex100/s5/local/data_prep.sh [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+## Only run this file from the example root directory
+##      $ ./local/data_prep.sh
+
+mkdir -p "data/train" "data/test" "data/local"
+
+source ./path.sh
+
+CORPUS_DIR="$1"
+
+N_SPEAKERS=100
```
Comment (danpovey) [inline egs/dimex100/s5/local/data_prep.sh]: Please clarify what these values mean, etc.

**Comment by alx741** [comment]
There we go. @danpovey please let me know if I got something wrong about those observations

**Comment by danpovey** [comment]
Can you please rename to egs/spanish_dimex100?  It will be more discoverable then.

**Comment by alx741** [comment]
> It will be more discoverable then.

Agreed

**Comment by danpovey** [comment]
Thanks!  Merging.

**Pull request merged**